CONTRIBUTING.rst

```diff
@@ -8,6 +8,7 @@ participating, you are expected to honor this code.
 Running the tests
 -----------------
 
+
```

Comment:
er why?

Reply:
nothing special, but first commit went unsuccessfully with coveio arguing  on diffs
I know this is a special case for codeio to bark the wrong tree, so another additional commit with just blank line forces it to rerun it successfully

Reply:
Oh, I you've separated the commits. Thank you very much!